environments/prod/inventory

```diff
@@ -18,3 +18,6 @@ hypothesis
 
 [box:children]
 hypothesis
+
+[papertrail:children]
+hypothesis
```

Comment:
We got rid of all the `:children` on the inventories in tutor-deployment. Not saying you have to do it, just FYI